core/src/main/java/com/klaytn/caver/methods/response/FeeHistory.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.methods.response;
+
+import org.web3j.protocol.core.Response;
+
+import java.util.List;
+
+public class FeeHistory extends Response<FeeHistory.FeeHistoryData> {
```

Comment:
Can we change name to `FeeHistoryResult`?

Reply:
No problem. I'll change name and also Data.

Reply:
I changed the name.
Thanks :)